packages/files-ui/src/Components/Modules/FileBrowsers/CreateOrManageSharedFolder.tsx

```diff
@@ -171,118 +422,227 @@ const CreateOrManageSharedFolder = ({ mode, onClose, bucketToEdit }: ICreateOrMa
         </Typography>
       </div>
       {mode === "create" &&
-          <div className={classes.modalFlexItem}>
-            <TextInput
-              className={classes.shareFolderNameInput}
-              labelClassName={classes.inputLabel}
-              label={t`Shared Folder Name`}
-              value={sharedFolderName}
-              onChange={onNameChange}
-              autoFocus
-              state={nameError ? "error" : "normal"}
-              data-cy="input-shared-folder-name"
-            />
-            {nameError && (
+        <div className={classes.modalFlexItem}>
+          <TextInput
+            className={classes.shareFolderNameInput}
+            labelClassName={classes.inputLabel}
+            placeholder={t`Shared Folder Name`}
+            label={t`Shared Folder Name`}
+            value={sharedFolderName}
+            onChange={onNameChange}
+            autoFocus
+            state={nameError ? "error" : "normal"}
+            data-cy="input-shared-folder-name"
+            size="large"
+          />
+          {nameError && (
+            <Typography
+              component="p"
+              variant="body1"
+              className={classes.errorText}
+            >
+              {nameError}
+            </Typography>
+          )}
+        </div>
+      }
+      <div
+        className={classes.userNameSuggest}
+        ref={ref}
+        onClick={() => { !searchActive && setSearchActive(true)}}
+      >
+        {mode === "create" && <Typography
+          className={classes.boldLabel}
+          component="p"
+        >
+          Add users to shared folder
```

Comment:
Translation is missing
```suggestion
          <Trans>Add users to shared folder</Trans>
```